File: Modul 5/NO 4/PRAK404-2210817310016-Riyo Aurora Guison.c
#include<stdio.h>
int main()
{
float a,b,c;
while(1){
printf("\nPilih Program\n");
printf("1.Penjumlahan\n");
printf("2.Pengurangan\n");
printf("3.Perkalian\n");
printf("4.Pembagian\n");
printf("5.Exit\n");
printf("Masukkan Pilihan :");
scanf("%f",&c);
if(c>5){
    printf("Input anda salah,silahkan coba lagi\n");
    continue;
}else if (c==5){
    printf("Terimakasih,telah menggunakan kalkulator Riyo Aurora Guison\n");
    break;
}
printf("Masukkan Nilai Pertama :");
scanf("%f",&a);
printf("Masukkan Nilai Kedua :");
scanf("%f",&b);
if(c==1){
    printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n",a,b,a+b);
}else if(c==2){
    printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n",a,b,a-b);
}else if(c==3){
    printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n",a,b,a*b);
}else if(c==4){
    printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n",a,b,a/b);
}
}
}

File: Modul 5/NO 4/PRAK404-2210817310016-Riyo Aurora Guison.py
while True:
    print("\nPilih Program\n 1.Penjumlahan\n 2.Pengurangan\n 3.Perkalian\n 4.Pembagian\n 5 Exit")
    c=int(input("Masukkan Pilihan ="))
    if c > 5 :
        print("Input anda salah, silahkan coba lagi")
        continue
    elif c==5:
        print("Terima kasih, telah menggunakan kalkulator Riyo Aurora Guison")
        break
    else:
        a=float(input("Masukkan Nilai Pertama="))
        b=float(input("Masukkan Nilai Kedua="))
        if c==1:
            print("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f"%(a,b,a+b))
        elif c==2:
            print("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f"%(a,b,a-b))
        elif c==3:
            print("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f"%(a,b,a*b))
        elif c==4:
            print("Hasil pembagian antara %.2f dengan %.2f adalah %.2f"%(a,b,a/b))